drivers/sensor/bme680/Kconfig

```diff
@@ -0,0 +1,161 @@
+# Kconfig.bme680 - BME680 temperature, pressure, humidity and gas sensor configuration options
+
+#
+# Copyright (c) 2016, 2017 Intel Corporation
+# Copyright (c) 2017 IpTronix S.r.l.
+# Copyright (c) 2018 Bosch Sensortec GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+menuconfig BME680
+	bool "BME680 sensor"
+	depends on I2C
+	default	n
+	help
+	 Enable driver for BME680 I2C-based based temperature, pressure, humidity and gas sensor.
+
+config BME680_DEV_NAME
+	string "BME680 device name"
+	depends on BME680
```

Comment:
It would be cleaner to get rid this and all of the below

"depends on BME680"

statements by wrapping the configs with an if statement.

```
if BME680

config BME680_DEV_NAME
[...]

endif # BME680

endmenu # BME680
```